index.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Asad Shareef-Portfolio</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+ 
+<!-- This is the header with navigation and other files -->
+
```

Comment:
Please do not leave unused code Unused code as comments make your code not clean.